packages/router/src/directives/router_link_active.ts

```diff
@@ -99,6 +108,7 @@ export class RouterLinkActive implements OnChanges, OnDestroy, AfterContentInit
    */
   @Input() routerLinkActiveOptions: {exact: boolean}|IsActiveMatchOptions = {exact: false};
 
+  @Output() public readonly routerLinkActivated: EventEmitter<boolean> = new EventEmitter();
```

Comment:
```suggestion
  @Output() readonly routerLinkActivated: EventEmitter<boolean> = new EventEmitter();
```
nit: `public` is not necessary